### PR TITLE
Removes sidebar update class

### LINKED ISSUE #630: resetButtons method on sidebar doesn't remove class
When resetting sidebar buttons the class remains - still can cause problems when doing advanced plugins in the sidebar

### PR DESCRIPTION
**Pull request by darylhedley**
Also stringifies options object so it can work with nodejs v0.12

fixes #630

### CODE REVIEW

File: plugins/content/course/index.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ function initialize () {
       var options = _.keys(req.body).length
       ? req.body
       : req.query;
+      var options = JSON.stringify(options);
```
Comment (finbartracey1) [inline plugins/content/course/index.js]: you could remove the var. Already declared in line above var options = _.keys(req.body).length

Comment (dennis-learningpool) [inline plugins/content/course/index.js]: @darylhedley - actually, can you remove this bit? I think this is related to the issues with node v0.12, but we're only supporting 0.10.33 at the moment, and this workaround needs a closer look.

**Comment by brian-learningpool** [comment]
+1

**Comment by finbartracey1** [comment]
+1

**Pull request merged**